z_infra_provisioning/cloud_infra_center/ocp_upi/down-network.yaml

```diff
@@ -65,11 +65,4 @@
   - name: 'List cluster networks'
     command:
       cmd: "openstack network list --tags {{ cluster_id_tag }} -f value -c Name"
-    register: networks
-
-  - name: 'Remove the cluster networks'
-    os_network:
-      name: "{{ item.1}}"
-      state: absent
-    with_indexed_items: "{{ networks.stdout_lines }}"
-    
\ No newline at end of file
+    register: networks    
```

Comment:
Since we don't destroy the network, so it's unnecessary to `list cluster networks`

Reply:
Yes, you're right, I've removed it too.